src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterNameAccessModifierChecker.java

```diff
@@ -0,0 +1,23 @@
+/*
+ParameterName
+format = ^h$
+ignoreOverridden = (default)false
+accessModifiers = \tpublic
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.parametername;
+
+public class InputParameterNameAccessModifierChecker {
```

Comment:
`InputParameterNameWhitespaceInAccessModifierProperty`, we have another tool called checker, let's avoid this term.

Reply:
Done.